helloworld/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+const arithmetic = require('./lib/arithmetic.js');
+
+module.exports = function(){
```

Comment:
This function is actually not getting invoked anywhere. Your console.logs at the bottom are only invoking variations of ```greet``` and ```arithmetic``` but never does this function at line 6 really happen.

In other words, if I type ```node index.js``` in my terminal, I only get back all the console.logs below, I don't actually get the return value that should be coming from this specific function. 

To solve that, I would have line 6 be: 

```
const main = module.exports = function(){
  \\\do stuff here
}
```

Then in the same module, I'd invoke ```main``` using ```console.log(main())```.